Doc/whatsnew/3.7.rst

```diff
@@ -169,6 +169,8 @@ Optimizations
   using the :func:`os.scandir` function.
   (Contributed by Serhiy Storchaka in :issue:`25996`.)
 
+* :meth:`selectors.DefaultSelector.modify` on Linux, OSX and Solaris is around
```

Comment:
"DefaultSelector" depends on the platform, I would prefer to mention concrete classes: PollSelector, EpollSelector and DevpollSelector.